frontend/package.json

```diff
@@ -7,8 +7,8 @@
 		"dev": "cross-env NODE_ENV=development webpack serve --progress",
 		"build": "webpack --config=webpack.config.prod.js --progress",
 		"prettify": "prettier --write .",
-		"lint": "eslint . --debug",
-		"lint:fix": "eslint . --fix --debug",
```

Comment:
can we add `--debug` back ?

Reply:
debug doesn't add any value, also it increases the time of running the command because of the debug output which doesn't add any value to us (as a user).

Reply:
yeah make sense.. but you can have one more pr as this is not relevant to trace detail page

Reply:
reverted 👍🏼